content/terms/how-to/apply-filters.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Apply filters
+weight: 7
+---
+
+# Apply filters
+
+This guide explains how to apply filters to existing declarations to remove meaningless content that changes on each page load or that cannot be removed with CSS selectors to avoid noise in the terms changes history.
+
+## Prerequisites
+
+- An existing terms declaration file
+- Identified the noise you want to remove and ensure it cannot be removed with CSS selectors with the [`remove`]({{< relref "terms/reference/declaration/#ref-remove" >}}) property.
```

Comment:
I will give an example of noise that will be filtered by `removeQueryParams`, the only built-in filter available at the moment.

Reply:
It's not a tutorial, it's an how to guide, so the idea is not to learn but to apply concrete solution to a real word problem, so I'm not sure it is appropriate